CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -215,6 +225,22 @@ local function createFrame(name, size, position, bgTransparency, bgColor)
 	return frame
 end
 
+local function createFrame3D(name, size, position, bgTransparency, bgColor)
+	local frame = Instance.new("ImageLabel")
+	frame.Name = name
+	frame.Size = size + UDim2.new(0, 2, 0, 2)
+	frame.Position = (position + UDim2.new(0, -1, 0, -1)) or UDim2.new(0, -1, 0, -1)
+	frame.BackgroundTransparency = 1
+	frame.ImageColor3 = bgColor or Color3.new()
+	frame.ImageTransparency = bgTransparency
+	frame.Image = "rbxasset://textures/ui/VR/rectBackgroundWhite.png"
+	frame.ScaleType = Enum.ScaleType.Slice
+	frame.SliceCenter = Rect.new(10, 10, 10, 10)
```

Comment:
I am just worried that a render code change could make the center section of these images not show up at all since there are no pixels in the center areas to sample from